src/com/mendix/widget/ProgressCircle/components/ProgressCircle.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, DOM } from "react";
+
+import * as classNames from "classnames";
+import { Circle } from "progressbar.js";
+
+import "../ui/ProgressCircle.css";
+
+export interface OnclickProps {
+    onClickType: ProgressOnclick;
+    microflowProps?: MicroflowProps;
```

Comment:
Removed. Wonder who came up with this MicroflowProps thing... removed it before I saw this.